src/widgets/dragAndDrop.ts

```diff
@@ -169,50 +183,50 @@ function makeDraggable (tr, obj) {
  **  successHandler(file, uploadedURI)    Called after each success upload
  **                              With file object an final URI as params
  */
-function uploadFiles (fetcher, files, fileBase, imageBase, successHandler) {
-  for (var i = 0; files[i]; i++) {
-    const f = files[i]
-    debug.log(
+export function uploadFiles (fetcher: any, files: any, fileBase: string, imageBase: string, successHandler: any) {
```

Comment:
A similar thing happened here with successHandler.  files as well.  I can put this as FileList which it is, but for testing I couldn't quickly figure this out.  I tried to createElement('input') and set type to 'files' but it didn't work. Fetcher I just realized, I will look at this.

Reply:
I tried to use Fetcher from rdflib, but it gave me the error .webOperation does not exist on type 'Fetcher'